### PR TITLE
I added the chaincode-seminar and the subfolders for Bitcoin and Lightning. In the folder chaincode-seminar/lightning-network, I added the trascript for th…

### PR DESCRIPTION
**Pull request by dulcedu**
…e Video: https://www.youtube.com/watch?v=toarjBSPFqI Title: Rene-pickhardt-onion-routing-with-htlcs.md

### CODE REVIEW

**Comment by dulcedu** [comment]
@adamjonas I made the changes you suggested and I created a new pull request since was easy for me.

**Comment by adamjonas** [comment]
I really don't want to slow you down or discourage your efforts, but what I meant by:

> I don't think this belongs in the chaincode residency directory though since it isn't related to chaincode or the residency. If you can't find a good home for it, there is always the misc folder.

... is that I really think the folder placement implies that credit should be attributed to chaincode when we had nothing to do with its creation. I'm not sure that Rene would have a problem with it but I think it's a bad precedent to set. Again, sorry for being pedantic, but I just think it's the right message.

**Comment by adamjonas** [comment]
ping @dulcedu

**Comment by adamjonas** [comment]
@dulcedu Following up here.

File: chaincode-labs/chaincode-seminars/lightning-network/2019-07-22-rene-pickhardt-onion-routing-with-htlcs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Rene Pickhardt - Onion Routing with HTLCs on the Lightning Network 
+Transcript_by: Dulce Villarreal
+categories: ['chaincode-seminars']
```
Comment (kouloumos) [inline chaincode-labs/chaincode-seminars/lightning-network/2019-07-22-rene-pickhardt-onion-routing-with-htlcs.md:4]: This category doesn't help much imo and also chaincode seminars resources might change in the future. I would agree with the categorization if maybe that's an indicator that this is quality material, but hopefully everything in here are considered quality resources. Tags are much better in categorization of the material.

File: chaincode-labs/chaincode-seminars/lightning-network/2019-07-22-rene-pickhardt-onion-routing-with-htlcs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Location: Part of “Hacking the Lightning Network - A protocol to scale Bitcoin” book project
+
+Date: Jan 22, 2019
+
+Video: <https://www.youtube.com/watch?v=toarjBSPFqI>
+
+Transcript by: Dulce Villarreal 
+
+
+## Introduction
```
Comment (kouloumos) [inline chaincode-labs/chaincode-seminars/lightning-network/2019-07-22-rene-pickhardt-onion-routing-with-htlcs.md:22]: Having only this header kind of indicates that this a non-stop Introduction. Maybe consider adding more sections or have no header at all?

**Comment by kouloumos** [comment]
Happy to review if there is future interest to continue the work on this.

**Pull request closed without merging**